docs/doxygen/migration/mqtt.txt

```diff
@@ -0,0 +1,127 @@
+/**
+@page mqtt_migration MQTT
+@brief How to migrate an MQTT application from v3 to 202009.00.
+
+The MQTT library has been redesigned in 202009.00 as a solely single-threaded library. New features such as persistent session support and QoS 2 publish delivery have also been added.
+
+The features of the MQTT libraries in v3 and 202009.00 are defined by the common MQTT 3.1.1 spec; therefore, the two versions are similar.
+
+@section migration_mqtt_removed Removed features
+@brief The following features were present in v3, but removed in 202009.00:
+- <u>Auto-reconnect</u> <br>
+  Version 3 has an auto-reconnect feature triggered through a call to `aws_iot_mqtt_yield`. In version 202009.00, `aws_iot_mqtt_yield` has been removed, as well as the auto-reconnect feature. <br>
```

Comment:
The second statement says that `aws_iot_mqtt_yield` has been removed but the last table of equivalent functions says that `MQTTProcessLoop` (and ReceiveLoop) are equivalent to it.

I suggest that the second statement purely mentions that the "re-connect" feature has been removed from the 202009.00 version of the MQTT library

Reply:
Removed